### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by KordonDev** [apollographql/graphql-tag#40]
I have to use the `null` as value in mutations. Therefore I cloned the repository and packaged it.

Now in `parse.js` line 488 in the function `parseValueLiteral` accepts null as value. But when I use it in the browser i get the error:
```
RangeError: Maximum call stack size exceeded
    at stripLoc (https://localhost:9000/lib/graphql-tag/browser.js:77:19)
    at stripLoc (https://localhost:9000/lib/graphql-tag/browser.js:105:26)
    at stripLoc (https://localhost:9000/lib/graphql-tag/browser.js:105:26)
    at stripLoc (https://localhost:9000/lib/graphql-tag/browser.js:105:26)
    at stripLoc (https://localhost:9000/lib/graphql-tag/browser.js:105:26)
    at stripLoc (https://localhost:9000/lib/graphql-tag/browser.js:105:26)
    at stripLoc (https://localhost:9000/lib/graphql-tag/browser.js:105:26)
    at stripLoc (https://localhost:9000/lib/graphql-tag/browser.js:105:26)
    at stripLoc (https://localhost:9000/lib/graphql-tag/browser.js:105:26)
    at stripLoc (https://localhost:9000/lib/graphql-tag/browser.js:105:26)
```

To use Graphql-tag in the browser I used browserify (`browserify index.js --standalone graphqlTag -o=browser.js`).

In my file line 77 is:`function stripLoc (doc, removeLocAtThisLevel) {`.

Line 105 is
```
if (valueType === '[object Object]' || valueType === '[object Array]') {
     doc[keys[key]] = stripLoc(value, true);
}
```
in function `stripLoc`.

**Comment by jnwng** [comment]
@KordonDev would it be possible to include the `gql` document that is triggering this? we shouldn't need a schema or anything, just the text that is getting parsed by `graphql-tag`.

**Comment by jnwng** [comment]
looks like this may be related: https://github.com/apollographql/graphql-tag/issues/6

**Comment by KordonDev** [comment]
I posted the library in a gist you can find [here](https://gist.github.com/KordonDev/2c293261e400c08bf1d86ce94e388d68). To generate the file I cloned the repository and transpiled it with the newest versions of the libraries in `package.json`.

My query looks like this
```
gql`
query {
    opportunities {
        name
    }
}
`
```

**Comment by KordonDev** [comment]
Further investigation got me to the point that I noticed that with the now graphql version, the property `loc` has to new properties `startToken` and `endToken`. Both are of the type `Tok`. `Tok` has the properties `next` and `previous` which point to another `Tok` with these two properties. This is the point where the infinite loop appears.

My workaround is to store the properties `startToken` and `endToken` in local variables and attach it after the for loop again.

I will make an pull request where you can see the problem.

**Comment by jnwng** [comment]
this should be resolved as part of `graphql-tag@v1.3.0`
